.github/workflows/container-image-buildah.yml

```diff
@@ -0,0 +1,100 @@
+name: Container Image
+
+on:
+  workflow_dispatch: {}
```

Comment:
This allows triggering a build manually for any branch/tag if so needed/desired.